buildchain/buildchain/deps.py

```diff
@@ -0,0 +1,36 @@
+from doit.exceptions import TaskError
```

Comment:
You need a docstring to describe what your module is doing, e.g.: `"""Tasks to check the dependencies on external commands."""` as the first line of the file